### ISSUE TITLE
cracklabs.com/prawto is down. Move or copy documentation to github

### ISSUE THREAD

**Issue by adamthedeveloper** [prior/prawnto#18]

**Comment by joeperic** [comment]
The only traces of documentation I've found is in Railscasts:
http://railscasts.com/episodes/153-pdfs-with-prawn

**Comment by lachlanjc** [comment]
Yep, the site is still down (apparently it was "recently registered on Namecheap"). Was there anything useful on the site?
